java-test-samples/apigw-lambda-ddb/src/test/java/com/example/TicketFunctionIntegrationTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT-0
+ */
+
+package com.example;
+
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.amazonaws.services.lambda.runtime.tests.annotations.Event;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import software.amazon.awssdk.auth.credentials.AwsCredentials;
+import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider;
+import software.amazon.awssdk.http.HttpStatusCode;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// This is an integration test as it requires an actual AWS account.
+// It assumes that a DynamoDB table with name "tickets" exists on AWS in US-EAST-1
+// and reads AWS Credentials from ~/.aws/credentials with default profile
+@ExtendWith(SystemStubsExtension.class)
+public class TicketFunctionIntegrationTest {
+
+  private final DynamoDbClient ddbClient = DynamoDbClient.builder()
+    .region(Region.US_EAST_1)
+    .build();
+  @SystemStub
+  private EnvironmentVariables environmentVariables;
+  private List<String> ticketList = new ArrayList<>();
+
+  @BeforeEach
```

Comment:
Do we need to do that each time? Is `@BeforeAll` possible instead?

Reply:
Fixed